algrtm/src/app/customSolutions/page.js

```diff
@@ -1,7 +1,32 @@
+import CustomSolutionsSection from "@/components/CustomSolutionsSection";
+import IconsColoredComponent from "@/components/IconsColoredComponent";
+import LinksSection from "@/components/LinksSection";
+import OfferSection from "@/components/OfferSection";
+import RoundedTextComponent from "@/components/RoundedTextComponent";
+
+import SolutionsSection from "@/components/SolutionsSection";
+import TechnologiesSection from "@/components/TechnologiesSection";
+import RoundedRectangle from "@/components/shared/RoundedRectangle";
 import React from "react";
 
 const CustomSolutions = () => {
-  return <div>Custom SOLUTIONS</div>;
+  return (
+    <main className="flex flex-col items-center justify-center px-[2.75rem]">
+      <CustomSolutionsSection />
+      <OfferSection />
+      <TechnologiesSection />
+      <RoundedTextComponent
+        backgroundColor="neutralLight-neutral10"
+        textTitleColor="neutralLight-neutral100"
+        textParagraph1Color="neutralLight-neutral60"
+        version="version2"
+        showButton={true}
+      />
+      <RoundedRectangle />
+      <SolutionsSection version="version2" />
+      <LinksSection />
```

Comment:
It seems to me that the LinkSection is the same on every page, so we can move it directly to `layout.js` so that it is not rerendered when the route changes